.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+### 🛠 Changes being made
+
+JIRA
+Description
+
+### 🧪 Test plan
+
+- [ ] Functionality tested manually.
+- [ ] Unit tests are added.
+- [ ] The documentation updated.
+
+### 🏎 Quality check
+
+- [ ] Perform self review before.
+- [ ] Data type and buffer overflows checked.
```

Comment:
```suggestion
- [ ] Does this change follow security best practices
```
I think we could start with maintaining the list of SDL objections for the review on wiki page and keep this checklist generic